tempesta_fw/http_msg.c

```diff
@@ -54,10 +54,9 @@ tfw_http_msg_hdr_val(TfwStr *hdr, int id, TfwStr *val)
 
 	TFW_STR_FOR_EACH_CHUNK(c, hdr, {
 		BUG_ON(!c->len);
-		BUG_ON(c->len >= val->len);
```

Comment:
Did you see crash on the assertion? Typically it should always be true. Just ">=" should be replaced by ">".